### ISSUE TITLE
Running error while installing

### ISSUE THREAD

**Issue by banro21** [krandiash/financial-tone#2]
```
`(env) ubuntu@ip-xxxx~/guardrails$ guardrails hub install hub://cartesia/financial_tone
2024-03-31 20:11:54 ip-xxxxxx guardrails-cli[7820] NOTICE Installing hub://cartesia/financial_tone...
  Running command git clone --filter=blob:none --quiet ...
.....
INFO Collecting git+https://github.com/krandiash/financial-tone.git
  Cloning https://github.com/krandiash/financial-tone.git to /tmp/pip-req-build-toevdg8f
  Resolved https://github.com/krandiash/financial-tone.git to commit 26b71c897f13900e2bc1d7ccaa89d086a31050db
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'done'
  Preparing metadata (pyproject.toml): started
  Preparing metadata (pyproject.toml): finished with status 'done'
Building wheels for collected packages: financial_tone
  Building wheel for financial_tone (pyproject.toml): started
  Building wheel for financial_tone (pyproject.toml): finished with status 'done'
  Created wheel for financial_tone: filename=financial_tone-0.0.0-py3-none-any.whl size=12524 sha256=05f854822a0b02e908539d617177039d3512e624ddd72ed6a07bcef2db5fd9b8
  Stored in directory: /tmp/pip-ephem-wheel-cache-wcldyd4e/wheels/0f/b4/d8/4672f892796b72d674501d53d506140583150705a6ac7acccd
Successfully built financial_tone
Installing collected packages: financial_tone
Successfully installed financial_tone-0.0.0

ERROR: unknown command "inspect"
2024-03-31 20:12:01 ip-172-31-86-75 guardrails-cli[7820] ERROR Failed to inspect `
```

What could be going wrong here?

**Comment by banro21** [comment]
Looks like the dependency is on pip inspect which is available only on version 22.2+

**Comment by banro21** [comment]
Closing the issue as i am able to go past the error.
